packages/server/src/models/Achievement.ts

```diff
@@ -0,0 +1,17 @@
+import { Model } from "objection";
+
+export class Achievement extends Model {
+  id!: number;
+  name!: string;
+  static tableName = "achievements";
```

Comment:
Move all table names to `models.ts` declaration. See details in the comments above.